sql3/test/defs/defs_date_functions.go

```diff
@@ -19,48 +22,56 @@ var datePartTests = TableTest{
 	),
 	SQLTests: []SQLTest{
 		{
+			name: "DatePartIncorrectParamsCount",
```

Comment:
Didn't know test case names can be defined. I merged a whole bunch of tests without names, will remember to fix them when I touch those files next time.